package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "nestjs",
+  "version": "1.0.0",
+  "description": "NestJS Architect example project",
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "lint": "eslint '{src,apps,libs,test}/**/*.ts' --fix",
+    "test:e2e": "echo 'No e2e tests implemented yet.'"
+  },
+  "dependencies": {
+    "@nestjs/common": "9.0.1",
+    "@nestjs/core": "9.0.1",
+    "@nestjs/platform-express": "9.0.1",
+    "@nestjs/serve-static": "3.0.0",
+    "@nestjs/typeorm": "^9.0.1",
+    "class-transformer": "0.5.1",
+    "class-validator": "0.14.0",
+    "pg": "^8.9.0",
+    "reflect-metadata": "0.1.13",
+    "rimraf": "3.0.2",
+    "rxjs": "7.5.5",
+    "typeorm": "^0.3.11"
+  },
+  "devDependencies": {
+    "@nestjs/cli": "9.0.0",
+    "@nestjs/schematics": "9.0.1",
+    "@nestjs/testing": "9.0.1",
+    "@types/express": "4.17.13",
+    "@types/node": "^18.0.3",
+    "@types/supertest": "2.0.12",
+    "@typescript-eslint/eslint-plugin": "5.30.5",
+    "@typescript-eslint/parser": "5.30.5",
+    "eslint": "8.19.0",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-import": "2.26.0",
+    "prettier": "2.7.1",
+    "supertest": "6.2.4",
```

Comment:
This module can probably be removed as well as `@nestjs/testing` and `@types/supertest`